op.c

```diff
@@ -16509,11 +16570,18 @@ Perl_subsignature_start(pTHX)
     Newx(signature, 1, yy_parser_signature);
     SAVEDESTRUCTOR_X(&destroy_subsignature_context, signature);
 
-    signature->elems    = 0;
-    signature->optelems = 0;
+    signature->next_argix = 0;
+    signature->opt_params = 0;
```

Comment:
Switch Newx() to Newxz() above instead of DIY zeroing. The authors of Foo LibC know malloc ptrs are aligned and the exact internal/secret alignment (which can be higher than type `__m128i` or `NV`) and can be zeroed in units of 4, 8, 16 (SSE) or 32 (AVX)  per CPU ins, so they can/may/could/do zero memory faster than you can, and def faster than memset() can.